ligo/main.mligo

```diff
@@ -212,6 +218,6 @@ let s = update_time_weighted_sum s in
  match p with
 | X_to_Y p -> x_to_y s p
 | Y_to_X p -> y_to_x s p
-| Set_position p -> set_position s p.lower_tick_index p.upper_tick_index p.lower_tick_witness p.upper_tick_witness p.liquidity_delta p.to_x p.to_y
+| Set_position p -> set_position s p
```

Comment:
:+1:

Reply:
Yeah I remember you suggesting this :smile: